test.sh

```diff
@@ -8,6 +8,23 @@
 
 set -e
 
+if [[ -f "./manifest.sh" ]]; then
+  echo "Files are generated by dist-gen. Check if everything generated by distgen properly."
+  echo "Generating all files by 'make generate-all' ..."
+  make generate-all
+  echo "Generated files are generated."
+  # common submodule is not updated at all. No argument to git diff is needed.
```

Comment:
Wondering about this comment - if I understand it correctly then the `git diff` command would show a diff for the common submodule if it was updated right? We do update the common submodule for PRs made against container-common-scripts that use the `Required-by` keyword in the commit message. Would this work with them as well?

Reply:
I don't think so. We do not do git pull in a submodule. We don't care about the submodule. This would be another test to test if the submodule is updated or not.
But I have found out the command which can be used for it.

`git diff --ignore-submodules=all`

Reply:
Example:
```bash
$ git diff --ignore-submodules --exit-code
$ echo $?
0
$ git diff --exit-code
diff --git a/common b/common
index ce7b9d1..77d1ea2 160000
--- a/common
+++ b/common
@@ -1 +1 @@
-Subproject commit ce7b9d1aa48cf65bb8071254e4e5396de410904b
+Subproject commit 77d1ea2246524f5849bc088e58d967739f1cb681
$ echo $?
1
$
```